renovate.json

```diff
@@ -107,11 +107,21 @@
         "task/validate-fbc/**"
       ]
     },
+    {
+      "groupName": "preflight-deprecated",
```

Comment:
Instead of having two groups, is it possible to utilize [ignorePaths](https://docs.renovatebot.com/configuration-options/#ignorepaths) to ignore the earlier versions?

Reply:
I don't see this being used anywhere right now, and this was the direction the maintainers told me to go. Is `ignorePaths` a top level attribute? Or can it be used in `packageRules`?

Reply:
I tried setting it in `packageRules` and tested with https://github.com/konflux-ci/build-definitions/pull/2219#pullrequestreview-2783178706, but it didn't seem to work